clients/binaryartifact.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clients
+
```

Comment:
> my understanding is that we want clients only for things that interface with services outside of our control, e.g. connecting to GitHub, to OSV database, etc. @azeemsgoogle is this correct?

I agree. I don't think an interface is needed here since there is no usecase to override it with different implementations.